.github/2.0/cookbooks/Executor.md

```diff
@@ -516,6 +516,61 @@ In 2.0rc1, the following fields are valid for `metas` and `runtime_args`:
 - for any other parametrization of the Executor, you can still access its constructor arguments (defined in the class `__init__`) and the request `parameters`
 - `workspace` will be retrieved from either `metas` or `runtime_args`, in that order
 
+### Handle parameters
+Parameters are passed to executors via `request.parameters` with `Flow.post(..., parameters=)`. This way all the `executors` will receive 
+`parameters` as an argument to their `methods`. These `parameters` can be used to pass extra information or tune the `executor` behavior for a
+given request without updating the general configuration.
+
+```python
+from typing import Optional
+from jina import Executor, requests, DocumentArray, Flow
+
+class MyExecutor(Executor):
+    def __init__(self, default_param: int = 1, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.default_param = default_param
+
+    @requests
+    def foo(self, docs: Optional[DocumentArray], parameters: dict, **kwargs):
+        param = parameters.get('param', self.default_param)
+        # param may be overriden for this specific request
+        assert param == 5
+
+with Flow().add(uses=MyExecutor) as f:
+    f.post(on='/endpoint', inputs=DocumentArray([]), parameters={'param': 5})
+```
+
+However, this can be a problem when the user wants different executors to have different values of the same parameters. 
+In that case one can specify specific parameters for the specific `executor` by adding a `dictionary` inside parameters with 
+the `executor` name as `key`. Jina will then take all these specific parameters and copy to the root of the parameters dictionary before 
```

Comment:
"Before calling the executor `method, Jina will take all these specific parameters and copy them to the root of the parameters dictionary`." 

Still super long sentence but don't know how to restructure it.

Reply:
```suggestion
Before calling the executor method, Jina will take all these specific parameters and copy them to the root of the parameters dictionary
```